src/dpq2/conv/time.d

```diff
@@ -84,6 +84,10 @@ if( is( T == SysTime ) )
     );
 }
 
+/++
+    Structure to represent PostgreSQL's Timestamp without time zone type.
+    Basically it works just like SysTime but allways assumed to be in UTC.
```

Comment:
> Basically it works just like SysTime but **allways assumed to be in UTC**.

False assumption!

Reply:
In my opinion comment about TZ assumption is more suitable in SysTime opCast method.

Reply:
And my be opCast method can have optional TZ parameter instead of forced UTC...

Reply:
...And, probably, opCast should be renamev to `toSysTime` because it.

Reply:
Good catch, I've misunderstood the docs.
I adjusted the description and renamed opCast to toSysTime with added parameter.
I also modified the behavior so it works with LocalTime instead and added some more tests for it.